instana-java-sdk/src/main/java/com/instana/sdk/annotation/Span.java

```diff
@@ -46,15 +46,19 @@
    *         {@link Object#toString()} method is invoked on all arguments. If any such invocation triggers an exception,
    *         the arguments for this methods are not stored. It is the responsibility of the user to assure that these
    *         methods do not imply any side-effects to the application.
+   * @deprecated use annotation {@code @TagParam }
```

Comment:
add ... "which gives you full control over the names of the tags to be created"

Reply:
done